tests/dragonfly/snapshot_test.py

```diff
@@ -125,21 +125,21 @@ async def test_dbfilenames(
 
 @pytest.mark.slow
 @dfly_args({**BASIC_ARGS, "dbfilename": "test-cron", "snapshot_cron": "* * * * *"})
-async def test_cron_snapshot(tmp_path: Path, async_client: aioredis.Redis):
+async def test_cron_snapshot(tmp_dir: Path, async_client: aioredis.Redis):
```

Comment:
my bad 😅  ... but why did it pass 🤔

Reply:
it did not. the slow tests are not running in regression tests